main-backend/utils/estimator.py

```
@@ -0,0 +1,32 @@
+def estimate_project(message: str):
+    message = message.lower()
+
+    if "gym website" in message or "fitness website" in message:
+        return {
+            "project": "Gym Website",
+            "time": "7–10 days",
```

Ruff (0.14.6)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)